File: app/src/main/java/knf/kuma/backup/screens/MigrateVersionFragment.java
package knf.kuma.backup.screens;

import android.content.Context;
import android.content.pm.PackageInfo;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import knf.kuma.R;

public class MigrateVersionFragment extends Fragment {

    @BindView(R.id.tv_version_bad)
    TextView version;

    public MigrateVersionFragment() {
    }

    public static int getInstalledCode(Context context) {
        try {
            PackageInfo info = context.getPackageManager().getPackageInfo("knf.animeflv", 0);
            return info.versionCode;
        } catch (Exception e) {
            return -1;
        }
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.lay_migrate_version, container, false);
        ButterKnife.bind(this, view);
        version.setText(String.valueOf(getInstalledCode(getContext())));
        return view;
    }
}

File: app/src/main/java/knf/kuma/backup/screens/MigrateVersionFragment.kt
package knf.kuma.backup.screens

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.core.content.pm.PackageInfoCompat
import androidx.fragment.app.Fragment
import knf.kuma.App
import knf.kuma.R
import org.jetbrains.anko.find

class MigrateVersionFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.lay_migrate_version, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        view.find<TextView>(R.id.tv_version_bad).text = installedCode.toString()
    }

    companion object {

        val installedCode: Long
            get() {
            return try {
                val info = App.context.packageManager.getPackageInfo("knf.animeflv", 0)
                PackageInfoCompat.getLongVersionCode(info)
            } catch (e: Exception) {
                -1
            }
        }
    }
}
